src/routers/baseFieldsRouter.ts

```diff
@@ -15,5 +15,10 @@ baseFieldsRouter.put(
 	requireAdministratorRole,
 	baseFieldsHandlers.putBaseField,
 );
+baseFieldsRouter.put(
```

Comment:
We also should allow for `GET /:id/localizations`, which would return all localizations for a given base field.